lib/math.js

```diff
@@ -1,37 +1,12 @@
 /*
-* Shortens a number to the abbreviated thousands, millions, billions, etc
-* https://stackoverflow.com/a/40724354
-
-* @param {number} number: value to shorten
-* @returns {string|number}
-
-* @example
-* // return '12.3k'
-* abbreviateNumber(12345, 1)
-*/
-
-const SI_PREFIXES = ['', 'k', 'M', 'G', 'T', 'P', 'E'];
-
-export const abbreviateNumber = (number, precision = 0) => {
-  // what tier? (determines SI prefix)
-  const tier = (Math.log10(number) / 3) | 0;
-
-  const round = value => {
-    return precision === 0 ? Math.round(value) : value.toFixed(precision);
-  };
-
-  // if zero, we don't need a prefix
-  if (tier === 0) {
-    return round(number);
+ * Adds commas to separate the digits of numbers (example: 100,000)
+ */
+export const formatCurrencyWithCommas = number => {
+  let locale = 'en-US';
+  if (typeof window !== 'undefined') {
+    locale = window.navigator.language;
```

Comment:
That's not a great way to detect the locale, we should ideally rely on `react-intl`'s `intl.locale`. Maybe `formatCurrencyWithCommas` should take the locale as a parameter?

Also, isn't `toLocaleString` automatically using the browser locale if not provided?

Reply:
@Betree : Sorry, I was confused that we have the detect the browser locale not the locale of open-collective. Anyways, I've just pushed a change to use the `intl.locale` instead and updated the above screen-cast as well. Let me know how it goes. 🙂